src/Network.cpp

```diff
@@ -914,6 +914,11 @@ void Network::fill_input_plane_pair(const FullBoard& board,
     }
 }
 
+float Network::get_normalised_komi(const GameState* const state) {
+    const auto norm_komi = 0.5f + (state->get_komi() / (2.0f * TRAINED_UNIT_KOMI));
+    return norm_komi;
```

Comment:
Why not just return the expression outright? Even for the sake of clarity, the identifier is completely redundant.